.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: [5.5, 5.6, 7.0, 7.1, 7.2, 7.3, 7.4]
+        php: [7.1, 7.2, 7.3, 7.4, 8.0]
```

Comment:
How about testing directly in 8.1 ?

Reply:
Ok, but I included only `{ php: 8.1, dependencies: "highest" }`. 
For some reason, it fails with the dependencies version lowest.

Reply:
I think we should not test that much versions of PHP.
First, it's bad for the planet
Second, testing only 7.4 and 8.1 is more than enough.

Reply:
Why not to test also 7.0 and 7.2?

Reply:
Because it's not supposed to be used already 😉

Reply:
There are still a lot of projects using unsupported versions of PHP.
However, I see your point. I will stop testing 7.2 and 7.3, and keep testing 7.1 to prevent breaking changes.

Reply:
Excellent, thanks for that mate!

Reply:
:)